java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -228,14 +238,20 @@ public static Scalar timestampDaysFromInt(Integer value) {
     return Scalar.timestampDaysFromInt(value.intValue());
   }
 
+  /**
+   * Returns a duration scalar based on the type parameter.
+   * @param type - dtype of scalar to be returned
+   * @param value - corresponding value for the scalar
+   * @return - Scalar of the respective type
+   */
   public static Scalar durationFromLong(DType type, long value) {
```

Comment:
The `durationFromLong(DType type, Long value)` method to create a null duration scalar is missing.

Reply:
Isn't that covered by `Scalar.fromNull(DType type)` method?

Reply:
I see `timestampFromLong` also has a similar method. I will update, thanks